pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   path: ">=1.6.4 <2.0.0"
   googleapis_auth: ^0.2.6
   googleapis: ^0.54.0
-  code_builder: ">=3.1.3 <3.3.0"
+  code_builder: "3.4.0"
```

Comment:
we either should use "^3.4.0" or ">=3.4.0 <=3.6.0" depending on compatibility. Mockito currenly has the frist one